### PR TITLE
Add toRep

### LINKED ISSUE #246: [generics-rep] Add a `toRep` function
```purescript
toRep :: forall from to. Generic from to => Proxy from -> Proxy to
toRep _ = Proxy
```

### PR DESCRIPTION
**Pull request by JordanMartinez**
Fixes #246

### CODE REVIEW

File: src/Data/Generic/Rep.purs [hunk 1]
```diff
@@ -34,3 +37,6 @@ newtype Argument a = Argument a
 class Generic a rep | a -> rep where
   to :: rep -> a
   from :: a -> rep
+
+toRep :: forall from to. Generic from to => Proxy from -> Proxy to
```
Comment (kl0tl) [inline src/Data/Generic/Rep.purs]: Could we reuse `a` and `rep` variables, like in the class declaration?

Comment (kl0tl) [inline src/Data/Generic/Rep.purs]: Also I don’t know about you but I frequently confuse `to` and `from`, and I’m sure having `to :: rep -> a` but `toRep :: Proxy a -> Proxy rep` is going to be even more confusing to me 😬 May I suggest `repOf` instead?

Comment (JordanMartinez) [reply src/Data/Generic/Rep.purs]: Sounds good to me!

Comment (JordanMartinez) [reply src/Data/Generic/Rep.purs]: Fixed.

**Review by kl0tl — APPROVED** [review]
Thank you!

**Review by thomashoneyman — APPROVED** [review]

**Pull request merged**